evm/src/executor/inspector/cheatcodes/fork.rs

```diff
@@ -152,7 +208,7 @@ fn select_fork<DB: DatabaseExt>(
     fork_id: U256,
 ) -> Result {
     if state.broadcast.is_some() {
-        return Err(Error::SelectForkDuringBroadcast)
+        return Err(Error::SelectForkDuringBroadcast);
```

Comment:
nit re: formatting

you might have format on save enabled—we use the nightly toolchain for clippy and rust, so to make the fmt/clippy jobs pass you might need to do `cargo +nightly fmt` and `cargo +nightly clippy --workspace` before you push.